src/Features/Core/Portable/AddImport/CodeActions/AssemblyReferenceCodeAction.cs

```diff
@@ -7,22 +7,21 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.CodeActions;
 using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.SymbolSearch;
 using Microsoft.CodeAnalysis.Tags;
+using Microsoft.CodeAnalysis.Text;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.CodeFixes.AddImport
 {
     internal abstract partial class AbstractAddImportCodeFixProvider<TSimpleNameSyntax>
     {
-        private partial class AssemblyReference
-        {
             private class AssemblyReferenceCodeAction : CodeAction
             {
-                private readonly AssemblyReference _reference;
+                private readonly ReferenceAssemblyWithTypeResult _referenceAssemblyWithType;
                 private readonly string _title;
                 private readonly Document _document;
-                private readonly SyntaxNode _node;
```

Comment:
Remove usage of compiler types.  We only want simple data types in the code actions.